test/forms/inputs/checkbox-group.test.js

```diff
@@ -125,3 +125,21 @@ test('CheckboxGroup passes down props to children', () => {
   expect(checkbox).not.toHaveClass('custom-group-class')
   expect(checkbox).toHaveClass('custom-input-class')
 })
+
+test('DropdownCheckboxGroup sets menu no longer active when clicked outside', async () => {
```

Comment:
Since this isn't a real component anymore, should this be "CheckboxGroup with dropdown = true" (or similar)?